core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/layout/withLayout.ts

```diff
@@ -0,0 +1,54 @@
+import { patchState, signalStoreFeature, type, withComputed, withMethods } from '@ngrx/signals';
+
+import { computed } from '@angular/core';
+
+import { DotLayout } from '@dotcms/dotcms-models';
+
+import { LayoutProps } from './models';
+
+import { mapContainerStructureToDotContainerMap } from '../../../utils';
+import { UVEState } from '../../models';
+
+/**
+ * Add computed properties to the store to handle the Layout UI
+ *
+ * @export
+ * @return {*}
+ */
+export function withLayout() {
```

Comment:
Does layout needs it own store?

Reply:
It could have its own store, but the only thing that it manage is the auto-save/prevent leaving thing